src/model.js

```diff
@@ -182,6 +182,74 @@ export class Seed extends Str {
       }
     }
   }
+
+  /**
+   * encrypt the password.
```

Comment:
The comment should follow the style like
```
strengthenPassword strengthens the given password by hashing the given rounds.
```
Please check & update any other similar issue in this PR.